### PR TITLE
Add questionary

### PR DESCRIPTION
**Pull request by apeltzer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ewels** [comment]
I'm happy to be listed as a maintainer! 🚀

File: recipes/questionary/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - questionary
+
+about:
+  home: https://github.com/tmbo/questionary
+  license: MIT
+  license_family: MIT
+license_file: LICENSE.txt
+  summary: 'Python library to build pretty command line user prompts'
+  description: |
+    Easy to use multi-select lists, confirmations, free text prompts ...
```
Comment (ewels) [inline recipes/questionary/meta.yaml:39]:
```suggestion
  description: "Easy to use multi-select lists, confirmations, free text prompts.."
```

**Comment by apeltzer** [comment]
I think this looks good now: @conda-forge/help-python for review :-)

**Review by ewels — APPROVED** [review]

File: recipes/questionary/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Copyright 2020 Tom Bocklisch and contributors
```
Comment (synapticarbors) [inline recipes/questionary/LICENSE.txt:1]: You can remove this file since it's in the source distribution

File: recipes/questionary/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - questionary
+
+about:
+  home: https://github.com/tmbo/questionary
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```
Comment (synapticarbors) [inline recipes/questionary/meta.yaml:38]: change to `license_file: LICENSE` to use the one in the source distribution

File: recipes/questionary/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f2999f01735db77a80d6cb119766cb15b84c468cab325168941a3e0d91207437
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/questionary/meta.yaml:18]: I think you can remove this `build` section

File: recipes/questionary/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    - prompt_toolkit >=2.0,<4.0
+    - python
+    - poetry >=1.0.5
+  host:
+    - python
```
Comment (synapticarbors) [inline recipes/questionary/meta.yaml:23]: Change to `python >=3.6,<3.10` here and under `run`.

File: recipes/questionary/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    - pip
+    - poetry >=1.0.5
+  run:
+    - python
+    - prompt_toolkit >=2.0,<4.0
+
+test:
+  imports:
+    - questionary
```
Comment (synapticarbors) [inline recipes/questionary/meta.yaml:32]: add `- questionary.prompts` to import test

**Review by synapticarbors — CHANGES_REQUESTED** [review]

File: recipes/questionary/LICENSE.txt (hunk 2 above)
Comment (ewels) [reply recipes/questionary/LICENSE.txt:1]:
```suggestion
```

File: recipes/questionary/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - prompt_toolkit >=2.0,<4.0
+    - python
+    - poetry >=1.0.5
```
Comment (ewels) [inline recipes/questionary/meta.yaml:21]:
```suggestion
```

File: recipes/questionary/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+  host:
+    - python
+    - pip
+    - poetry >=1.0.5
+  run:
+    - python
```
Comment (ewels) [inline recipes/questionary/meta.yaml:27]:
```suggestion
    - python >=3.6,<3.10
```

File: recipes/questionary/meta.yaml (hunk 6 above)
Comment (ewels) [reply recipes/questionary/meta.yaml:32]:
```suggestion
    - questionary
    - questionary.prompts
```

File: recipes/questionary/meta.yaml (hunk 5 above)
Comment (ewels) [reply recipes/questionary/meta.yaml:23]:
```suggestion
    - python >=3.6,<3.10
```

File: recipes/questionary/meta.yaml (hunk 3 above)
Comment (ewels) [reply recipes/questionary/meta.yaml:38]:
```suggestion
  license_file: LICENSE
```

**Review by ewels — COMMENTED** [review]
Made suggested changes comments so that it's easy for you to apply @apeltzer

**Comment by apeltzer** [comment]
All addressed @synapticarbors

File: recipes/questionary/LICENSE.txt (hunk 2 above)
Comment (synapticarbors) [reply recipes/questionary/LICENSE.txt]: Just to clarify, I meant that you an remove the entire license file from within the recipe directory of this package.

Comment (ewels) [reply recipes/questionary/LICENSE.txt]: Sorry, I don't think @apeltzer or I realised this at first - now deleted! 🗑️

**Review by synapticarbors — APPROVED** [review]

**Comment by apeltzer** [comment]
So there is something odd with the windows tests that worked before - could you restart these @synapticarbors ? Thank you :-)

**Comment by synapticarbors** [comment]
@apeltzer -- just restarted it. Let's see what happens.

**Comment by ewels** [comment]
Thank you for the help @synapticarbors!

**Pull request merged**